client/store/helpers/mutations.js

```diff
@@ -30,6 +30,10 @@ export const save = (state, model) => {
   Vue.set(state.items, model.uid, model);
 };
 
+export const saveAll = (state, models) => {
```

Comment:
Consider casting model on save to array and have save do what is now part of save & saveAll